main.js

```diff
@@ -1 +1,19 @@
 
+function dis (value)
+{
+document.getElementById("edu").value+=value
+;}
+
+function solve()
+{
+let x = document.getElementById("edu").value
+let y = eval(x)
+document.getElementById ("edu").value = y
+}
+
+function clr()
+{
+document.getElementById("edu").value = ""
+
+
+}
```

Comment:
Great work on this, Mike! Code is nice and clean. Love how you've been partnering up and to become a better programmer. It's a really great skill to have and I see your growth!